### PR TITLE
fixed report layout

### PR DESCRIPTION
**Pull request by Dragos0000**

### CODE REVIEW

File: resources/eds_templates/diff_report/config.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
     "template": "main.html",
     "conf":
     {
-        "default_endpoint" : "http://localhost:3030/cob/sparql",
+        "default_endpoint" : "http://localhost:3030/report1/sparql",
```
Comment (cosletmihai) [inline resources/eds_templates/diff_report/config.json:5]: this is a redundant change as this serves only as an example, the data from there not being used.

File: resources/eds_templates/diff_report/templates/layout.html [hunk 2]
```diff
@@ -9,68 +9,123 @@
...
+    <link href="https://cdn.datatables.net/1.10.22/css/jquery.dataTables.min.css" rel="stylesheet" type="text/css">
+
+
+    <style>
+        .buttons-print {
+            margin-bottom: 0.5em !important;
+        }
+
+        #toc {
+            width: 15%;
```
Comment (cosletmihai) [inline resources/eds_templates/diff_report/templates/layout.html:28]: you should put the `css` into this [file](https://github.com/meaningfy-ws/rdf-differ/blob/master/rdf_differ/entrypoints/ui/static/main.css)

File: resources/eds_templates/diff_report/templates/macros.html [hunk 3]
```diff
@@ -6,7 +6,7 @@
             <div class="header">
                 Error
             </div>
-            <p>{{ message }}</p>
+            <p>{{ message|e }}</p>
```
Comment (cosletmihai) [inline resources/eds_templates/diff_report/templates/macros.html:9]: what's `e`?

**Review by cosletmihai — APPROVED** [review]

File: resources/eds_templates/diff_report/templates/layout.html (hunk 2 above)
Comment (Dragos0000) [reply resources/eds_templates/diff_report/templates/layout.html:28]: maybe in the future when eugen wants to

**Pull request merged**